supportedBackends/local/src/test/resources/application.conf

```diff
@@ -0,0 +1,10 @@
+local-config {
+  root = "local-cromwell-executions"
+  filesystems = {
+    local {
+      localization: [
+        "hard-link", "soft-link", "copy"
+      ]
+    }
+  }
+}
```

Comment:
I see `LocalBackendSpec` that you're parsing a similar config stanza manually, so I was just wondering if this was actually used